scss/_patterns_buttons.scss

```diff
@@ -1,70 +1,53 @@
-// Default blockquote styling
-@mixin vf-buttons {
-  @include vf-primary-button;
-  @include vf-secondary-button;
-}
+// Default button styling
+@mixin vf-p-buttons {
+
+  .button {
+    padding: .75rem 1.5rem;
 
-// Primary button styling
-@mixin vf-primary-button {
-  button,
-  input[type='button'],
-  input[type='reset'],
-  input[type='submit'],
-  .button--primary {
-    @include vf-button($color-light, $color-light-grey, null);
+    &--neutral {
+      @extend .button;
```

Comment:
Have you checked the CSS generated by this? Wonder if a placeholder would perform better.